src/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
Fix formatting to get right indentations to pass tests :) 

```suggestion
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
  </head>
  <body>
    Hello, world!
  </body>
</html>
```